docs/vectorizer-quick-start.md

```diff
@@ -123,7 +123,7 @@ To create and run a vectorizer, then query the auto-generated embeddings created
     ```sql
     SELECT
         chunk,
-        embedding <=>  ai.openai_embed('text-embedding-3-small', 'pgai', dimensions=>768) as distance
+        embedding <=>  ai.openai_embed('text-embedding-3-small', 'good food', dimensions=>768) as distance
```

Comment:
If you change the query like this, I'm pretty sure the results table isn't true anymore 😁  there is a blog post about food in there...